docs/UserGuide.md

```diff
@@ -4,14 +4,15 @@
 ## Contents
 * [Introduction](#introduction)
 * [Quick Start](#quick-start)
+* [Understanding the Command Syntax](#understanding-the-command-syntax)
 * [Features](#features)
    * [Managing Your Expenses](#managing-your-expenses)
    * [Managing Your Recurring Payments](#managing-your-recurring-payments)
    * [Managing Your Incomes](#managing-your-incomes)
    * [Managing Your Targets](#managing-your-targets)
    * [Handling Your Data](#handling-your-data)
    * [Listing all available commands: `help`](#listing-all-available-commands-help)
-   * [Exiting the application: `exit`](#exiting-the-application-exit)
+   * [Exiting the application: `exit`](#exiting-the-application-bye)
```

Comment:
Should it be "Exiting the application: `bye`" ?

Reply:
Thanks for catching that! I will make the change!